WalletWasabi.Fluent/ViewModels/Settings/BitcoinTabSettingsViewModel.cs

```diff
@@ -54,7 +56,7 @@ public BitcoinTabSettingsViewModel()
 				.Subscribe(_ => Save());
 		}
 
-		public Version BitcoinCoreVersion => Constants.BitcoinCoreVersion;
+		public string BitcoinCoreVersion => GetNodeVersion();
```

Comment:
This property, supposedly, calls `GetNodeVersion` every time the settings dialog is opened. This in turn allows us to have always a correct bitcoind version (i.e. one can terminate `bitcoind` and run a newer version and we would report a correct new value).

I don't think it can noticably slow down displaying of the settings page. If it somehow can, we should probably resort to caching.

Reply:
> I don't think it can noticably slow down displaying of the settings page. If it somehow can, we should probably resort to caching.

I don't think its required.